imagetest/test_suites/image_boot/setup.go

```diff
@@ -43,12 +49,12 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return nil
 	}
 
-	vm3, err := t.CreateTestVM("vm3")
+	vm4, err := t.CreateTestVM("vm4")
```

Comment:
is there some reason we need a completely separate VM just for this test?

Reply:
To isolate this test to run separately not sure how other other test would effect the boot time.

Reply:
Added the test in the same VM as secure boot as it does not affect the boot time